codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketUtil.java

```diff
@@ -31,11 +31,14 @@
  */
 final class WebSocketUtil {
 
+    // Suppress a warning about weak hash algorithm since it's defined in draft-ietf-hybi-thewebsocketprotocol-00
+    @SuppressWarnings("lgtm[java/weak-cryptographic-algorithm]")
     private static final FastThreadLocal<MessageDigest> MD5 = new FastThreadLocal<MessageDigest>() {
         @Override
         protected MessageDigest initialValue() throws Exception {
             try {
                 //Try to get a MessageDigest that uses MD5
+                //
```

Comment:
This empty comment looks like it was added by accident.

Reply:
My bad! I've removed this empty comment, thanks!